calicoctl/commands/node/diags.go

```diff
@@ -235,3 +243,39 @@ func writeDiags(cmds diagCmd, dir string) {
 		log.Errorf("Error writing diags to file: %s\n", err)
 	}
 }
+
+func walkIPTables(file string, command string)([]byte) {
+
+	tablesFile, err := os.Open(file)
+	if err != nil {
+		fmt.Printf("Failed to open: %s\n", file)
+	}
+
+	command, err = exec.LookPath(command)
+	if err != nil {
+		fmt.Printf("Failed to run command: %s\nError: Not found in PATH\n", command)
+	}
+
+	scanner := bufio.NewScanner(tablesFile)
+	scanner.Split(bufio.ScanLines)
```

Comment:
Is this just setting the split function to be the default? If so, do we need this line at all?